.github/workflows/build_workflow.yml

```
@@ -80,7 +80,7 @@ jobs:
           path: ~/conda_pkgs_dir
           key:
             ${{ runner.os }}-${{ matrix.python-version }}-conda-${{ env.CACHE_NUMBER }}-${{
-            hashFiles('spec-file.txt,setup.cfg') }}
+            hashFiles('spec-file.txt,pyproject.toml,.flake8') }}
```

I think we don't need the `.flake8` here because what this is trying to react to is whether the dependencies of `mache` have changed.
```suggestion
            hashFiles('spec-file.txt,pyproject.toml,') }}
```